agimus_controller/agimus_controller/ocp/ocp_croco_goal_reaching.py

```diff
@@ -9,7 +9,7 @@
 class OCPCrocoGoalReaching(OCPBaseCroco):
     def create_running_model_list(self) -> list[crocoddyl.ActionModelAbstract]:
         running_model_list = []
-        for _ in range(self._ocp_params.horizon_size - 1):
```

Comment:
Can you make it use the varying time steps please?
You can use list `self._ocp_params.timesteps`.